### PR TITLE
Create hearing types per organization

### LINKED ISSUE #928: admins can customize hearing_types per casa_org
Part of epic #880

AS an **admin**, 
I want to add/edit/show/hide `hearing_types` specific to my `casa_org`, 
so that only relevant `hearing_types` display for users within my organization. 

**Where does/should this occur?**
Under "Edit Organization" 

**How to access in QA**
<a href="https://casa-qa.herokuapp.com/casa_orgs/2/edit">Log in to QA</a> with the following credentials:
email: casa_admin1@example.com
password: 123456

**Acceptance Criteria**
- [ ] On the Edit CASA Organization page, there is a section called "Manage Court Details" (check to make sure this has not already been handled by #929)
In this section, an admin can: 
- [ ] create new `hearing_types`
- [ ] edit existing `hearing_types`
- [ ] hide `hearing_types`
- [ ] view hidden `hearing_types`
- [ ] show hidden `hearing_types`
- [ ] changes admins make here are reflected in `hearing_types` available to `supervisors`

### PR DESCRIPTION
**Pull request by haydenrou**
This is my first attempt to hit the acceptance criteria for the attached issue. Please let me know if it falls short at any point. 
Before this is ready to merge I had two questions:
1. Would you like seeding for hearing types, if so for which environments, and what hearing type names? Reference: https://github.com/rubyforgood/casa/issues/928#issuecomment-703990749
2. The last point on the acceptance criteria is " changes admins make here are reflected in hearing_types available to supervisors" - I don't understand what I need to do for this.

I plan on submitting a separate PR for #927 but wanted to get this first iteration out beforehand.

### What github issue is this PR for, if any?
Resolves #928 

### What changed, and why?
There is now a crud(except delete) for hearing types accessible via the casa org edit page. Hearing types can be flagged as active or inactive.

### How is this tested? (please write tests!) 💖💪
Via the edit org page, you can add a new hearing type or press the pencil to edit the hearing type.

### Screenshots please :)
hearingtypes

### CODE REVIEW

File: app/views/casa_orgs/edit.html.erb [hunk 1]
```diff
@@ -36,3 +36,11 @@
     <%= render "contact_types" %>
   </div>
 </div>
+
+<h1 class="pt-5">Manage Court Details</h1>
+
+<div class="card card-container">
+  <div class="card-body">
+    <%= render "hearing_types" %>
```
Comment (compwron) [inline app/views/casa_orgs/edit.html.erb:44]: Please add another test that the data shows up on the UI :)

**Review by compwron — APPROVED** [review]
This is looking very promising! 

- Please add another test for the data showing on the UI
- Please fix the CI build :)

Feel free to drop in to chat in slack #casa https://rubyforgood.herokuapp.com/ for a faster response to questions :)

**Comment by haydenrou** [comment]
These tests seem to be working locally for me, however some of the system ones seem to be somewhat flaky. Is there a way to retry CI without pushing up a change?

@compwron thanks for the feedback, I have added some specs.

@Zrrrpy looking forward to hearing your feedback here if you're happy with the approach, and what route you want to take regarding the seed data. If there is no conclusive answer yet, we could always add a backlog item and fill these in when we have better information.

When this has been approved and merged I'll make a start on #927

**Comment by compwron** [comment]
@haydenrou the flaky build should be fixed now! You can usually re-run the build by clicking on the build and clicking re-run but I don't see the button right now. O.o Pushing a new change (or a rebase to master) will work.

**Comment by compwron** [comment]
it's a pain to rebase/merge master when you're working on a fork instead of a branch. I moved your code to a branch but didn't open the PR- click here to open a new PR with all the code from this PR (plus a merge of master) so that you can get the hacktoberfest opening-PR credit :) https://github.com/rubyforgood/casa/compare/create-hearing-types?expand=1

**Comment by compwron** [comment]
Very cool! Sorry for the delay on merging this. 

Yes, seeding for hearing types would be great! For now only in dev/qa/staging, not in production. Be wary of the current discussion in slack about how we will change/combine/parameterize the seed files

**Pull request merged**